docs/Project.toml

```
@@ -3,4 +3,4 @@ Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 HypothesisTests = "09f84164-cd44-5f33-b23f-e6b0d136a0d5"
 
 [compat]
-Documenter = "~0.20"
+Documenter = "0.23"
```

This is the recommended form: https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#.travis.yml-Configuration-1
```suggestion
Documenter = "~0.23"
```

---

I am checking with Documenter.jl if they are in fact using patch versions for breaking changes during 0 major releases. If they are following the standard semver it should be the caret specifier (the default one).

---

Whatever the reason, I think we'd better do what the docs recommend. IIRC they indeed broke compatibility recently, so we can expect them to do so in the future too.

---

The docs have been updated to use the default semver caret instead of tilde (https://github.com/JuliaDocs/Documenter.jl/pull/1105).